src/l1-resources/resource-policy/README.md

```diff
@@ -3,7 +3,7 @@
 The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
 
 > **NOTE**: 
-> - **Atlas Resource Policies functionality is currently in Preview, you'll need to enable `Atlas Resource Policies` Beta Feature in your organization to use it.** 
+> - **Resource Policies is currently in Private Preview, if you want to test it, contact [MongoDB Support](https://www.mongodb.com/services/support) to enable `Atlas Resource Policies` Beta Feature in your organization.**
```

Comment:
not sure about the `test` word here

Reply:
maybe `try` is better?

Reply:
yes, but I'd wait for docs team to confirm :)

Reply:
I agree that `try` sounds better here.

Reply:
I think sticking to the prior language, maybe `use` would be better here.

Reply:
UPdated in 121a10d0be1e1183f4d1536886ec2fcf93bd765e.
WDYT @kyuan-mongodb @marcosuma ?